src/client/sandbox/node/index.js

```diff
@@ -101,4 +107,31 @@ export default class NodeSandbox extends SandboxBase {
         this.win.attach(window);
         this.element.attach(window);
     }
+
+    static processSelector (selector) {
+        const ATTRIBUTE_SELECTOR_REG_EX = /\[([\w-^]+)=(.+?)]/g;
+
+        selector = String(selector);
+
+        if (!ATTRIBUTE_SELECTOR_REG_EX.test(selector))
+            return selector;
+
+        return selector.replace(ATTRIBUTE_SELECTOR_REG_EX, (str, name, value) => {
+            var isAttributeValueBeginWithSelector = endsWith(name, '^');
```

Comment:
`isAttributeValueBeginWithSelector ` this is completely confusing. First of all, more likely it's `isAttrNameEndsWithSelector`. The best way will be to modify the re: 
```js
/\[([\w-^]+)(^?=)(.+?)]/g
```
You will have additional group in the middle - the `operator`. You can use it then to restore the selector.